alf/algorithms/agent.py

```diff
@@ -13,10 +13,9 @@
 # limitations under the License.
 """Agent for integrating multiple algorithms."""
 
-import functools
 from typing import Callable
 
-import gin.tf
+import gin
 import tensorflow as tf
```

Comment:
L41  Maybe its more clear that `Agent ` inherit from  `RLAlgorithm` instead of `OnPolicyAlgorithm`

Reply:
It's indeed OnPolicyAlgorithm because it can be trained using OnPolicyDriver. You can put comments related to Agent to this PR #200